integration/test/Test/MLS.hs

```diff
@@ -138,6 +139,7 @@ testMixedProtocolUpgrade secondDomain = do
     resp.status `shouldMatchInt` 200
     resp.json %. "protocol" `shouldMatch` "mixed"
     resp.json %. "epoch" `shouldMatchInt` 0
+    resp.json %. "receipt_mode" `shouldMatchInt` 7
```

Comment:
so, mixed conversations don't disable their receipt mode by force?  isn't it a problem to wait for that one conv member that's on vacation, or that got fired six months ago?

Reply:
What do you mean? As long as the conversation is mixed, it's operating as proteus, so I don't see a problem with leaving read receipts on. We discussed about this in the ticket.